ray-operator/controllers/ray/common/pod.go

```diff
@@ -25,6 +25,9 @@ const (
 	RayLogVolumeMountPath       = "/tmp/ray"
 	AutoscalerContainerName     = "autoscaler"
 	RayHeadContainer            = "ray-head"
+	objectStoreMemoryKey        = "object-store-memory"
```

Comment:
Let's also use `UpperCamelCase` for const definition?

Reply:
Ah I meant to ask if you meant to export the constants above. I thought I'd keep `objectStoreMemoryKey` unexported but can uppercase to be consistent with the rest.

Reply:
done in https://github.com/ray-project/kuberay/pull/332/commits/635ca71a36ad4da67b24a84163ebca523e8a05f6